apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -280,6 +285,17 @@ type IronicSection struct {
 	Template ironicv1.IronicSpec `json:"template,omitempty"`
 }
 
+// ManilaSection defines the desired state of Manila service
+type ManilaSection struct {
+	// +kubebuilder:validation:Optionala
+	// +kubebuilder:default=true
```

Comment:
Should we default Manila to false?

Reply:
Yes, makes sense as it's Optional: also, I have to fix a couple of typos in this patch when I'm done with my testing